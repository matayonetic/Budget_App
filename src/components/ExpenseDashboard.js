import React from "react";

// Dashboard
const ExpenseDashboard = () => {
  return (
    <div>
      <h2>Home Page</h2>
    </div>
  );
};

export default ExpenseDashboard