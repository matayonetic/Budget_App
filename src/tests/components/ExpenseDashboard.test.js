import ExpenseDashboard from "../../components/ExpenseDashboard";

test("", ()=>{})