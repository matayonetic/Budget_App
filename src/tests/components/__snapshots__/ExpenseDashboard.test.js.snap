// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render expense dashboard 1`] = `
<div>
  <ExpensesSummary />
  <Connect(ExpenseListFilter) />
  <ExpenseList />
</div>
`;
